src/common/resize-watcher.js

```diff
@@ -0,0 +1,77 @@
+//@flow
+import {FakeEvent, FakeEventTarget} from '@playkit-js/playkit-js';
+
+class ResizeWatcher extends FakeEventTarget {
+  _observer: Object;
+  _config: Object;
+
+  constructor(config: Object) {
+    super();
+    this._config = config;
+    this._initObserver();
+  }
+
+  destroy() {
+    this._observer.disconnect();
+    this._config = {};
```

Comment:
You passing the kaltura player config by reference and clean it here? I don't think this is valid...